uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
What does this actually look like? Wouldn't it include things we don't want, like kick-in levels? That's more an issue for Prime than for Magstock.

I'm still hoping to take this further by making pre-reg and at-door reg the same page. We need at-door reg to support registering/paying for multiple people at once, and I need the pre-reg page to save people's registrations before they're done paying.